pyproject.toml

```diff
@@ -189,7 +189,7 @@ line-ending = "auto"
 known-first-party = ['mirascope', 'tests', 'examples']
 
 [tool.pyright]
-exclude = ["mirascope/beta"]
+exclude = ["mirascope/beta", "**/bedrock/official*.py"]
```

Comment:
hmm, can we make this more specific to the examples? seems like that's what you mean to ignore?

Reply:
I wantted to ignore these files because they require excessive type hinting for dictionaries. This is likely because too many types would make the code unnecessarily long.